docs/advanced-stream.md

```diff
@@ -1,1344 +1,131 @@
 # Arcade Advanced Live
 
-Join the MakeCode developers as we stream MakeCode Arcade live on mixer.com/makecode. Weekdays at 1:00pm PST
+Join the MakeCode developers as we stream MakeCode Arcade live on twitch.tv/msmakecode. Weekdays at 1:00pm PST
 
-## Videos
+## Games
 
 ### ~ codecard
 
-* name: Making a survival game
-* description: Join Richard, Shannon, Daryl, and Joey as they create another randomly generated game idea! Today we started making a survival game and show how to make a game HUD.
-* youTubeId: SvXeJevUkb4
-* youTubePlaylistId: PLMMBk9hE-SeoHWvTpdezyL9JfF0tocJUT
-* imageUrl: /static/advanced-stream/SvXeJevUkb4.jpg
+* name: Boss Rush
+* description: Battle a series of boss fights in this boss rush style game! Can you defeat them all? You probably can't beat Daryl, he's pretty tough.
+* cardType: forumUrl
+* url: https://forum.makecode.com/t/arcade-advanced-stream-38-boss-rush-finale/2158
+* imageUrl: /static/livestream-games/boss-rush.png
 
 ---
 
-* name: Character Animations
-* description: Join Richard, Shannon, Joey, and Daryl as they add walking and sword swinging animations to an adventure game hero.
-* youTubeId: zRewjteisLw
-* youTubePlaylistId: PLMMBk9hE-SeoHWvTpdezyL9JfF0tocJUT
-* imageUrl: /static/advanced-stream/zRewjteisLw.jpg
+* name: Tony Albatross: Pro Dolphin
+* description: The pro dolphin lifestyle is all about sick stunts and nobody does it better than Tony. Gain speed underwater to launch yourself into the air and press A or B to rack up points.
+* cardType: forumUrl
+* url: https://forum.makecode.com/t/arcade-advanced-stream-169-tony-albatross-pro-dolphin-pt-4/5078/5
+* imageUrl: /static/livestream-games/tony-albatross.png
 
 ---
 
-* name: Character Animations
-* description: Join Richard, Shannon, Joey, and Daryl as they add walking and sword swinging animations to an adventure game hero.
-* youTubeId: zRewjteisLw
-* youTubePlaylistId: PLMMBk9hE-SeoHWvTpdezyL9JfF0tocJUT
-* imageUrl: /static/advanced-stream/zRewjteisLw.jpg
+* name: Spinning Plates
+* description: Pack a dishwasher as efficiently as possible by rotating dishes in this hectic puzzle game. Try to fit as many as you can!
+* cardType: forumUrl
+* url: https://forum.makecode.com/t/arcade-advanced-stream-128-spinning-dishes/3885/3
+* imageUrl: /static/livestream-games/spinning-plates.png
 
 ---
 
-* name: Boss Battle
-* description: Join Richard, Shannon, and Daryl as they make a game with an intense boss battle!
-* youTubeId: 19AO3H_lpuE
-* youTubePlaylistId: PLMMBk9hE-SeoHWvTpdezyL9JfF0tocJUT
-* imageUrl: /static/advanced-stream/19AO3H_lpuE.jpg
+* name: Hamlet
+* description: Y'know, it's Hamlet. With mice I guess? Seems pretty accurate to me. Oh and you control a spotlight so make sure you keep it on the actors.
+* cardType: forumUrl
+* url: https://forum.makecode.com/t/arcade-advanced-stream-155-hamlet-act-2/4537/5
```

Comment:
very small so feel free to ignore, but I think having this number will scroll people to the specific reply in the post? But it's also not bad to have to scroll up

Reply:
Yeah, should drop the trailing post id, if they're going to the forum post they probably want the youtube video / extended description and not a comment ~